x/twap/listeners_test.go

```diff
@@ -48,8 +49,8 @@ func (s *TestSuite) TestAfterPoolCreatedHook() {
 			denoms := osmoutils.CoinsDenoms(tc.poolCoins)
 			denomPairs0, denomPairs1 := types.GetAllUniqueDenomPairs(denoms)
 			expectedRecords := []types.TwapRecord{}
-			for i := 0; i < len(denomPairs0); i++ {
-				expectedRecord, err := types.NewTwapRecord(s.App.GAMMKeeper, s.Ctx, poolId, denomPairs0[i], denomPairs1[i])
+			for i := len(denomPairs0); i < 0; i-- {
+				expectedRecord, err := twap.NewTwapRecord(s.App.GAMMKeeper, s.Ctx, poolId, denomPairs0[i], denomPairs1[i])
```

Comment:
Why are we iterating in reverse order?

Reply:
okay, fixed in most recent commit. Will need to write in spec about these guarantees still though